src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -7,6 +7,7 @@ import { Button } from '@uswitch/trustyle.button'
 import { Icon } from '@uswitch/trustyle.icon'
 import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
 import WireFrameCell from '@uswitch/trustyle.wireframe-cell'
+import { Eligibility } from '@uswitch/trustyle.information-panel'
```

Comment:
i cant see where this import is used?

Reply:
It's just changing the old eligibility with the imported one. It is used as it was used before at the bottom of the table.